### PR TITLE
docs(samples): add try/catch to delete operations in testing

### LINKED ISSUE #17: CDN key functions: should get a Google CDN key failed
This test failed!

To configure my behavior, see [the Flaky Bot documentation](https://github.com/googleapis/repo-automation-bots/tree/main/packages/flakybot).

If I'm commenting on this issue too often, add the `flakybot: quiet` label and
I will stop commenting.

---

commit: c144a9cccc04d4a9fe99baa048822b86e5e9336c
buildURL: [Build Status](https://source.cloud.google.com/results/invocations/d852f211-9f2c-4287-bed2-e5a971b8bd73), [Sponge](http://sponge2/d852f211-9f2c-4287-bed2-e5a971b8bd73)
status: failed
<details><summary>Test output</summary><br><pre>Command failed: node getCdnKey.js long-door-651 us-central1 nodejs-test-stitcher-google-key-bdc9d1c6
5 NOT_FOUND: Resource 'projects/long-door-651/locations/us-central1/cdnKeys/nodejs-test-stitcher-google-key-bdc9d1c6' was not found

Error: Command failed: node getCdnKey.js long-door-651 us-central1 nodejs-test-stitcher-google-key-bdc9d1c6
5 NOT_FOUND: Resource 'projects/long-door-651/locations/us-central1/cdnKeys/nodejs-test-stitcher-google-key-bdc9d1c6' was not found

    at checkExecSyncError (child_process.js:635:11)
    at execSync (child_process.js:671:15)
    at Context.<anonymous> (test/stitcher.test.js:189:20)
    at processImmediate (internal/timers.js:461:21)</pre></details>

### LINKED ISSUE #18: CDN key functions: should delete a Google CDN key failed
This test failed!

To configure my behavior, see [the Flaky Bot documentation](https://github.com/googleapis/repo-automation-bots/tree/main/packages/flakybot).

If I'm commenting on this issue too often, add the `flakybot: quiet` label and
I will stop commenting.

---

commit: c144a9cccc04d4a9fe99baa048822b86e5e9336c
buildURL: [Build Status](https://source.cloud.google.com/results/invocations/d852f211-9f2c-4287-bed2-e5a971b8bd73), [Sponge](http://sponge2/d852f211-9f2c-4287-bed2-e5a971b8bd73)
status: failed
<details><summary>Test output</summary><br><pre>Command failed: node deleteCdnKey.js long-door-651 us-central1 nodejs-test-stitcher-google-key-bdc9d1c6
5 NOT_FOUND: Resource 'projects/long-door-651/locations/us-central1/cdnKeys/nodejs-test-stitcher-google-key-bdc9d1c6' was not found

Error: Command failed: node deleteCdnKey.js long-door-651 us-central1 nodejs-test-stitcher-google-key-bdc9d1c6
5 NOT_FOUND: Resource 'projects/long-door-651/locations/us-central1/cdnKeys/nodejs-test-stitcher-google-key-bdc9d1c6' was not found

    at checkExecSyncError (child_process.js:635:11)
    at execSync (child_process.js:671:15)
    at Context.<anonymous> (test/stitcher.test.js:197:20)
    at processImmediate (internal/timers.js:461:21)</pre></details>

### PR DESCRIPTION
**Pull request by irataxy**
Fixes b:202972969
Fixes: #17 
Fixes #18

### CODE REVIEW

File: samples/test/stitcher.test.js [hunk 1]
```diff
@@ -84,12 +84,16 @@ before(() => {
...
-          }
-        );
+        try {
+          execSync(
+            `node deleteSlate.js ${projectId} ${location} ${this.nextId}`,
+            {
+              cwd,
+            }
+          );
+        } catch (err) {
```
Comment (alclee-google) [inline samples/test/stitcher.test.js]: Do we need to check that the err is the NotFound error?

Comment (sofisl) [reply samples/test/stitcher.test.js]: Plus one, we should make sure we're not catching all errors

**Review by sofisl — CHANGES_REQUESTED** [review]

**Comment by sofisl** [comment]
@irataxy, will these close #17 and #18?

File: samples/test/stitcher.test.js (hunk 1 above)
Comment (irataxy) [reply samples/test/stitcher.test.js]: Refactored to ignore NOT_FOUND

**Review by sofisl — APPROVED** [review]

File: samples/test/stitcher.test.js [hunk 2]
```diff
@@ -84,15 +80,26 @@ before(() => {
...
+        ) {
+          try {
+            execSync(
+              `node deleteSlate.js ${projectId} ${location} ${this.nextId}`,
+              {
+                cwd,
+              }
+            );
+          } catch (err) {
+            if (err.message.includes('NOT_FOUND')) {
```
Comment (alclee-google) [inline samples/test/stitcher.test.js:97]: Does the error object have a status to check the http status code? Might be cleaner than checking the message.

**Review by alclee-google — APPROVED** [review]

**Pull request merged**